iOS/buildAndRunCIT.command

```diff
@@ -13,7 +13,7 @@ testLogPath=../$path/testLog.txt
 echo "******Build framework************" 2>&1 | tee -a $buildLogPath
 cd WindowsAzureMessaging
 xcodebuild clean &> /dev/null
-xcodebuild -scheme Framework -target Framework -configuration Release BUILD_DIR=./Build | sed '/setenv/d' 2>&1 | tee -a $buildLogPath
+xcodebuild BITCODE_GENERATION_MODE=bitcode -scheme Framework -target Framework -configuration Release BUILD_DIR=./Build | sed '/setenv/d' 2>&1 | tee -a $buildLogPath
```

Comment:
I don't think you'd want `BITCODE_GENERATION_MODE=bitcode` for the SDK. What you want is to set `BITCODE=NO` in the project and include the flags above.

Reply:
Okay, some more context on `BITCODE_GENERATION_MODE=bitcode` can be found in https://stackoverflow.com/questions/34959767/whats-the-difference-between-fembed-bitcode-and-bitcode-generation-mode. If I understand the way the SDK is built, the code above should work fine.

Reply:
I never used this flag `BITCODE_GENERATION_MODE` but it seems redundant with the project settings.